frontends/p4/modelInstances.cpp

```diff
@@ -16,10 +16,12 @@ limitations under the License.
 
 #include "coreLibrary.h"
 #include "fromv1.0/v1model.h"
+#include "common/model.h"
 
 namespace P4 {
 
 P4CoreLibrary P4CoreLibrary::instance;
+V2Model V2Model::instance;
```

Comment:
You cannot have the front-end depend on any model.
V1Model is there just for P4-14 compatibility.